### DISCUSSION TITLE
Can't Login GitHub on Mobile App and Browser

### DISCUSSION THREAD

**Discussion by IllusoryZephyr** [community/community#121528] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't login to GitHub using Kiwi Browser when it verifies with 2FA. After entering 2FA, the page says "Your browser did something unexpected" as shown in the following screenshots. First screenshot is when logging in directly in browser, second screenshot is when logging in through GitHub mobile app. 

I disabled all extensions and re-login, still doesn't work. Tried to login using Google Chrome instead, but by then I'm already rate-limited, so no result for now. 

Android 13
MIUI Global 14.0.5
Kiwi Browser 124.0.6327.2

![Screenshot_2024-04-29-19-13-47-948_com kiwibrowser browser](https://github.com/community/community/assets/58404312/3ab03a43-8224-48bb-8a6a-ddc2cc4e1866)
![Screenshot_2024-04-29-19-18-01-609_com kiwibrowser browser](https://github.com/community/community/assets/58404312/09c5117e-ce6d-4bfb-a362-2a1d7c5f5a7a)

**Comment by stevepopovich** [comment]
hi @IllusoryZephyr sorry to hear this is happening. Are you trying to log into the native mobile app, or the mobile website? Just wanna clarify here

**Reply by IllusoryZephyr** [reply]
Technically, it is the mobile website. Because logging into the native app requires logging through the website.

**Reply by stevepopovich** [reply]
Right, that's what I was asking for. Thanks for that. I will change this to the proper area of responsibility, but we on the native mobile team will keep an eye on this too.

**Reply by AuTsing** [reply]
I have the same problem, is there any solution?

**Reply by Alynva** [reply]
I have a similar problem. I have multiple browsers installed, after I hit "Login", it asks me which browser I want to open, which I suppose would show the "approve in your phone" message page and sometimes the two-digit code, but after I select the browser I want to open, it doesn't open anything, nor redirect the page to processed to login.
I do receive the notification in the app about the login request, but approving or rejecting it does nothing...

MIUI 14
Chrome 129

**Reply by tabbea** [reply]
@Alynva Yes, that's exactly the problem I have on Chrome browser too.

**Reply by zkingboos** [reply]
> I have a similar problem. I have multiple browsers installed, after I hit "Login", it asks me which browser I want to open, which I suppose would show the "approve in your phone" message page and sometimes the two-digit code, but after I select the browser I want to open, it doesn't open anything, nor redirect the page to processed to login.
> I do receive the notification in the app about the login request, but approving or rejecting it does nothing...
> 
> MIUI 14
> Chrome 129

Have you fixed it? I can't login through my phone on GitHub anymore

**Reply by KrzysztofadamBarden-TomTom** [reply]
Having similar issue, trying to login in on a mobile browser, mi or chrome, it shows a pop-up to open another browser (android intent?) and does nothing. GitHub native app does ask for the digits

**Comment by lyswhut** [comment]
Same problem, I got it back to work by switching to the desktop site to log in and then switching back to the mobile version

**Comment by IRNCyber** [comment]
I understand the frustration of being unable to log into GitHub using Kiwi Browser due to 2FA issues. Here are a few things you can try to resolve the problem:
1. Clear Browser Cache and Cookies:
 * Go to Kiwi Browser's settings and clear the browsing data, including cache and cookies.
 * This can sometimes resolve issues caused by corrupted data.
2. Try Incognito Mode:
 * Open an incognito window in Kiwi Browser and try logging in again.
 * This can help isolate issues caused by extensions or browser settings.
3. Update Kiwi Browser:
 * Make sure you're using the latest version of Kiwi Browser.
 * Updates often include bug fixes and security improvements.
4. Use a Different Browser:
 * If the issue persists with Kiwi Browser, try using a different browser like Google Chrome or Firefox.
 * This can help you determine if the problem is specific to Kiwi Browser or a broader issue with your device or network.
5. Contact Kiwi Browser Support:
 * Reach out to Kiwi Browser's support team and describe the issue in detail.
 * They may be able to provide specific troubleshooting steps or identify a known issue.
6. Check GitHub's Status Page:
 * Ensure there are no ongoing issues with GitHub's services that might be affecting logins.
7. Enable 2FA App-Based Verification:
 * If possible, consider enabling 2FA app-based verification instead of SMS or email.
 * This can be a more secure and reliable method of authentication.
If none of these solutions work, you may need to wait for a fix from Kiwi Browser or GitHub. In the meantime, you can try using a different browser or device to access your GitHub account.

**Reply by lyswhut** [reply]
This happens after entering the 2FA codes.
Unfortunately, Kiwi Browser is less actively maintained now, but you can still download the last version to reproduce it: https://github.com/kiwibrowser/src.next/releases

**Reply by IRNCyber** [reply]
It seems you're facing issues with 2FA authentication on Kiwi Browser. Here are some steps to troubleshoot and resolve the issue:

### 1. **Verify the 2FA Codes**
   - Ensure the time on your 2FA device (phone or app) is synced accurately to prevent mismatches.
   - If using an app like Google Authenticator or Authy, check if the codes are being updated in real-time.

### 2. **Update or Switch Browser**
   - Since Kiwi Browser is less actively maintained, consider using a more updated browser like Chrome or Firefox to ensure compatibility with modern web standards and security protocols.
   - If you must use Kiwi Browser, download the latest available version from the [GitHub releases page](https://github.com/kiwibrowser/src.next/releases).

### 3. **Clear Browser Data**
   - Clear cookies, cache, and other stored data for the website or application requiring 2FA to eliminate potential conflicts.

### 4. **Check for Extensions**
   - Disable browser extensions temporarily, as some may interfere with the 2FA process.

### 5. **Test on Another Device**
   - If possible, try completing the 2FA process on another browser or device to confirm the issue is browser-specific.

If the problem persists even after these steps, it might indicate compatibility issues with Kiwi Browser itself. Using a maintained browser is recommended to avoid such issues.

Let me know if you'd like more tailored advice! 😊
